js/npm/etherscan/README.md

```diff
@@ -0,0 +1,34 @@
+# @parity/Etherscan
+
+A thin, lightweight promise wrapper for the api.etherscan.io/apis service, exposing a common endpoint for use in JavaScript applications.
+
+[https://github.com/ethcore/parity/js/src/3rdparty/etherscan](https://github.com/ethcore/parity/js/src/3rdparty/etherscan)
```

Comment:
This should be `https://github.com/ethcore/parity/tree/master/js/src/3rdparty/etherscan`
Plus, don't we want to add the README there, so users can read it navigating to this URL ?